### DISCUSSION TITLE
Get-PnPProvisioningTemplate is not recognized

### DISCUSSION THREAD

**Discussion by ravinleague** [pnp/powershell#329] [General] [answered]
Hi,

I am getting an error Get-PnPProvisioningTemplate is not recognized when trying to export a site as a template.
I can use other APIs such as Connect-PnPOnline, get-pnplistitem etc..

Any help would be greatly appreciated

**Comment by gautamdsheth** [comment, marked as answer]
The correct command to use is `Get-PnPSiteTemplate` for exporting site templates.
You can read the docs for the same here - https://pnp.github.io/powershell/cmdlets/Get-PnPSiteTemplate.html
